{
  "schema_version": "1.6.2",
  "id": "BIT-pytorch-2025-3121",
  "summary": "PyTorch torch.jit.jit_module_from_flatbuffer memory corruption",
  "details": "A vulnerability classified as problematic has been found in PyTorch 2.6.0. Affected is the function torch.jit.jit_module_from_flatbuffer. The manipulation leads to memory corruption. Local access is required to approach this attack. The exploit has been disclosed to the public and may be used.",
  "aliases": [
    "CVE-2025-3121"
  ],
  "affected": [
    {
      "package": {
        "ecosystem": "Bitnami",
        "name": "pytorch",
        "purl": "pkg:bitnami/pytorch"
      },
      "severity": [
        {
          "type": "CVSS_V4",
          "score": "CVSS:4.0/AV:L/AC:L/AT:N/PR:L/UI:N/VC:N/VI:N/VA:L/SC:N/SI:N/SA:N/E:X/CR:X/IR:X/AR:X/MAV:X/MAC:X/MAT:X/MPR:X/MUI:X/MVC:X/MVI:X/MVA:X/MSC:X/MSI:X/MSA:X/S:X/AU:X/R:X/V:X/RE:X/U:X"
        }
      ],
      "ranges": [
        {
          "type": "SEMVER",
          "events": [
            {
              "introduced": "2.6.0"
            },
            {
              "fixed": "2.7.0"
            }
          ]
        }
      ]
    }
  ],
  "database_specific": {
    "severity": "Medium",
    "cpes": [
      "cpe:2.3:a:linuxfoundation:pytorch:*:*:*:*:*:python:*:*"
    ]
  },
  "references": [
    {
      "type": "WEB",
      "url": "https://github.com/pytorch/pytorch/issues/149800"
    },
    {
      "type": "WEB",
      "url": "https://github.com/pytorch/pytorch/issues/149800#issue-2940240700"
    },
    {
      "type": "WEB",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2025-3121"
    },
    {
      "type": "WEB",
      "url": "https://vuldb.com/?ctiid.303012"
    },
    {
      "type": "WEB",
      "url": "https://vuldb.com/?id.303012"
    },
    {
      "type": "WEB",
      "url": "https://vuldb.com/?submit.525049"
    }
  ],
  "published": "2025-05-28T11:55:44.258Z",
  "modified": "2025-05-28T12:14:51.629Z"
}